Riot/Modules/Room/RoomViewController.m

```diff
@@ -1248,40 +1248,65 @@ - (void)sendTextMessage:(NSString*)msgTxt
 {
     // The event modified is always fetch from the actual data source
     MXEvent *eventModified = [self.roomDataSource eventWithEventId:customizedRoomDataSource.selectedEventId];
+    
+    [self setupRoomDataSourceToResolveEvent:^(MXKRoomDataSource *roomDataSource) {
```

Comment:
Maybe add a comment to say that we do this in order to display the event properly when editing/replying from a non-live timeline, otherwise LGTM 👍